tests/unit/adapters/rest-adapter/build-query-test.js

```diff
@@ -14,6 +14,13 @@ test("buildQuery() returns an empty query when snapshot has no query params", fu
 });
 
 if (isEnabled('ds-finder-include')) {
+  test("buildQuery - doesn't fail without a snapshot", function(assert) {
+    const adapter = DS.RESTAdapter.create();
+    const query = adapter.buildQuery();
+
+    assert.deepEqual(query, {}, 'returns an empty query');
+  });
```

Comment:
I think we want to move this test outside of the `isEnabled('ds-finder-include')` check.